validation/jinja2/validation/_base.html

```diff
@@ -54,7 +54,7 @@
   </nav>
 
 
-  <main role="main" class="container">
+  <main role="main" class="main_container">
```

Comment:
Incidentally, `<main role="main">` is, not only redundant in writing, but also redundant in meaning. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#landmark

```suggestion
  <main class="main_container">
```

Reply:
Ah, good to know. I can take out the `role` part 👍